packages/alert/index.js

```diff
@@ -1,35 +1,15 @@
+// TODO: replace text-14 with a token
```

Comment:
The thing we need to do about the hard-coded classes is not to replace them with a token but to import them from `@warp-ds/component-classes`. This way they are included in a list of component classes that is later feeded to unocss configuration's `safelist`. That makes the engine generate css for those classes at build step, which we then can pass to components which are nested within shadow DOM. Otherwise those two classes would be missing.

On another note, `text-14` is not going to be replaced with a token, which I understand may have been unclear 😄  It's a utility class that unocss will generate respective css for, and because we have recently made both `text-{number}` and `text-{t-shirt-sizes}` generate respective font-size and line-height css values, this will work. We can have a chat about it so I can shed some more light on this 😊 

Finally, it would be great if we could add TODO comments right above related lines of code, if possible 🙏